src/schemas/21-526EZ/schema.js

```diff
@@ -1,9 +1,425 @@
+import _ from 'lodash/fp';
+
+import constants from '../../common/constants';
+import definitions from '../../common/definitions';
+
+// Common definition doesn't have bank name
+const directDepositDef = _.set('properties.bankName', { type: 'string' }, definitions.bankAccount);
+
+// Until we can use the common address definition...
+// Grab countries and state lists
+const countries = constants.countries.map(object => object.value);
+const countriesWithStateList = Object.keys(constants.states).filter(x => _.includes(countries, x));
+const countryStateProperties = _.map(constants.states, (value, key) => ({
+  properties: {
+    country: {
+      type: 'string',
+      'enum': [key]
+    },
+    state: {
+      type: 'string',
+      'enum': value.map(x => x.value)
+    }
+  }
+}));
+countryStateProperties.push({
+  properties: {
+    country: {
+      not: {
+        type: 'string',
+        'enum': countriesWithStateList
+      }
+    },
+    state: {
+      type: 'string',
+      maxLength: 51 // Is this right?
+    }
+  },
+});
+
+
+const addressDef = {
+  type: 'object',
+  oneOf: countryStateProperties, // holds country and state
+  properties: {
+    addressLine1: {
+      type: 'string'
+      // maxLength: 51 // Until we know otherwise?
+    },
+    addressLine2: {
+      type: 'string'
+      // maxLength: 51 // Until we know otherwise?
+    },
+    addressLine3: {
+      type: 'string'
+      // maxLength: 51 // Until we know otherwise?
+    },
+    city: {
+      type: 'string'
+      // maxLength: 51 // Until we know otherwise?
+    },
+    zipFirstFive: {
+      type: 'number',
+      minLength: 5,
+      maxLength: 5
+    },
+    zipLastFour: {
+      type: 'number',
+      minLength: 4,
+      maxLength: 4
+    },
+    militaryPostOfficeTypeCode: {
+      type: 'string'
+      // maxLength: 51 // Until we know otherwise?
+    },
+    militaryStateCode: {
+      type: 'string'
+      // maxLength: 51 // Until we know otherwise?
+    },
+    type: {
+      type: 'string',
+      enum: [
+        // Are these exact?
+        'international',
+        'domestic',
+        'military'
+      ]
+    }
+  }
+};
+
+// Apparently we don't need a suffix here.
+const fullNameDef = _.omit('properties.suffix', definitions.fullName);
+
 let schema = {
   $schema: 'http://json-schema.org/draft-04/schema#',
   title: 'SUPPLEMENTAL CLAIM FOR COMPENSATION (21-526EZ)',
   type: 'object',
-  definitions: {},
-  properties: {},
+  definitions: {
+    directDeposit: directDepositDef,
+    date: definitions.date,
+    // dateRange: definitions.dateRange // hopefully we can use this later
+    phone: definitions.phone,
+    fullPhone: {
+      type: 'object',
+      properties: {
+        areaNbr: {
+          type: 'string'
+          // Validation?
+        },
+        phoneNbr: {
```

Comment:
I requested that abbreviations be removed and EVSS agreed.  `areaNbr` ---> `areaNumber` (or `areaCode`???) & `phoneNbr` ---> `phoneNumber`

Reply:
Excellent! What about `efctvDt`?

Reply:
Judging by the field `pointOfContactAreaCode`, I think it'd make sense to change `areaNumber` to `areaCode`, but I'll wait until we get confirmation.